requirements-dev.in

```diff
@@ -2,5 +2,6 @@ django
 six
 tox
 pytest
+pytest-cov
```

Comment:
I'm working on this in a separate PR, so this will just lead to lots of merge conflicts. Would you mind removing this?

Reply:
Sure no probs, I just added it to check if my additions were all covered. I will remove it now.